src/dev/run_integrate_locale_files.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { run } from './run';
+import { integrateLocaleFiles } from './i18n/integrate_locale_files';
+
+run(async ({ flags: { path } }) => {
+  if (!path) {
```

Comment:
__question:__ what do you think if we forbid multiple `--path` at this point with a corresponding error message? I can't think of any use case where we may need to have multiple files to integrate at once... But let me know what you think